bin/check_tron_jobs.py

```diff
@@ -45,6 +44,13 @@ def parse_cli():
         default=None,
         help="Check a particular job. If unset checks all jobs",
     )
+    parser.add_argument(
+        "--run-interval",
+        help="Run interval for this monitoring script, default is 300 seconds",
```

Comment:
You can use %(default)s instead of hardcoding the default, it will get the value passed in below.

Also, can we make it more clear that this doesn't actually control how often the script runs? Maybe you can add something like, "Used to calculate re-alerting settings."

And add the units (seconds) to the help string.